plugins/api-docs/dist/manifest.js

```diff
@@ -284,5 +262,114 @@ exports.default = {
                 ],
             },
         },
+        {
+            name: "@genesislcap/pbc-reporting-ui",
+            enabled: true,
+            src: {
+                img_dir: "docs/img",
+                readme: "./README.md",
+            },
+            output: {
+                directory: "./docs/05_components/reporting",
+                img_dir: "docs/img",
+                readme: "foundation-reporting.mdx",
+                keywords: ["web", "frontend", "ui", "micro-front-ends", "reporting"],
+                tags: ["web", "frontend", "ui", "micro-front-ends", "reporting"],
+                pages: [
+                    {
+                        title: "Foundation Reporting",
+                        sidebar_label: "Foundation Reporting",
+                        id: "foundation-reporting",
+                    },
+                ],
+            },
+        },
+        {
+            name: "@genesislcap/grid-pro",
+            enabled: true,
+            src: {
+                api_docs: "./docs/api",
+                img_dir: "docs/img",
+                readme: "./README.md",
+            },
+            output: {
+                directory: "./docs/04_web/14_grids/01_grid-pro",
+                api_docs: "docs/api",
+                readme: "01_grid-pro.mdx",
+                keywords: ["genesis", "foundation", "ui", "grid", "grid-pro", "ag"],
+                tags: [
+                    "connected",
+                    "datasource",
+                    "grid",
+                    "grid-pro",
+                    "ag",
+                ],
```

Comment:
Doesn't necessarily need to be done in this PR but I am aware some feedback we've been getting from the docs is that it's difficult to search for things. I am thinking due to that we should try add as much relevent bits to these as possible to try and aid searchability. Won't block the merge for this for now, but equally if you can think of more ideas then that would be ideal. Same with the other new two blocks you've added